mjpc/tasks/humanoid/tracking/task.xml

```diff
@@ -0,0 +1,164 @@
+    <numeric name="agent_planner" data="3" />
+    <numeric name="agent_horizon" data="0.5" />
+    <numeric name="agent_timestep" data="0.004" />
+    <numeric name="sampling_spline_points" data="16" />
+    <numeric name="sampling_exploration" data="0.15" />
+    <numeric name="sampling_trajectories" data="16" />
+    <numeric name="gradient_spline_points" data="5" />
+    <numeric name="ilqg_num_rollouts" data="6" />
+    <numeric name="ilqg_regularization_type" data="1" />
+    <numeric name="ilqg_representation" data="2" />
+    <numeric name="cma_trajectories" data="16" />
+  </custom>
+
+  <default>
+    <default class="mocap_site">
+      <site type="sphere" size="0.027" rgba="0 0 1 1" group="2" material="target"/>
+    </default>
+  </default>
+
+  <worldbody>
+    <body name="mocap[pelvis]" mocap="true">
+      <site name="mocap[pelvis]" class="mocap_site" size="0.05" />
+    </body>
+    <body name="mocap[head]" mocap="true">
+      <site name="mocap[head]" class="mocap_site" />
+    </body>
+    <body name="mocap[ltoe]" mocap="true">
+      <site name="mocap[ltoe]" class="mocap_site" />
+    </body>
+    <body name="mocap[rtoe]" mocap="true">
+      <site name="mocap[rtoe]" class="mocap_site" />
+    </body>
+    <body name="mocap[lheel]" mocap="true">
+      <site name="mocap[lheel]" class="mocap_site" />
+    </body>
+    <body name="mocap[rheel]" mocap="true">
+      <site name="mocap[rheel]" class="mocap_site" />
+    </body>
+    <body name="mocap[lknee]" mocap="true">
+      <site name="mocap[lknee]" class="mocap_site" />
+    </body>
+    <body name="mocap[rknee]" mocap="true">
+      <site name="mocap[rknee]" class="mocap_site" />
+    </body>
+    <body name="mocap[lhand]" mocap="true">
+      <site name="mocap[lhand]" class="mocap_site" />
+    </body>
+    <body name="mocap[rhand]" mocap="true">
+      <site name="mocap[rhand]" class="mocap_site" />
+    </body>
+    <body name="mocap[lelbow]" mocap="true">
+      <site name="mocap[lelbow]" class="mocap_site" />
+    </body>
+    <body name="mocap[relbow]" mocap="true">
+      <site name="mocap[relbow]" class="mocap_site" />
+    </body>
+    <body name="mocap[lshoulder]" mocap="true">
+      <site name="mocap[lshoulder]" class="mocap_site" />
+    </body>
+    <body name="mocap[rshoulder]" mocap="true">
+      <site name="mocap[rshoulder]" class="mocap_site" />
+    </body>
+    <body name="mocap[lhip]" mocap="true">
+      <site name="mocap[lhip]" class="mocap_site" />
+    </body>
+    <body name="mocap[rhip]" mocap="true">
+      <site name="mocap[rhip]" class="mocap_site" />
+    </body>
+  </worldbody>
+
+  <sensor>
+    <user name="control" dim="21" user="3 0 0.0 0.1 0.3" />
+    <user name="joint vel." dim="27" user="0 0 0.0 0.1" />
+
+    <user name="root pos error" dim="3" user="6 30.0 0.0 100.0 0.1" />
+    <user name="head pos error" dim="3" user="6 30.0 0.0 100.0 0.1" />
+    <user name="ltoe pos error" dim="3" user="6 30.0 0.0 100.0 0.1" />
```

Comment:
TODO: to make the usage a bit easier, maybe aggregate left and right weights into a single slider?